quickwit-cli/src/cli.yaml

```diff
@@ -12,17 +12,25 @@ subcommands:
                 about: Creates an index.
                 display_order: 1
                 args:
-                    - metastore-uri:
-                        about: Location of the metastore.
-                        long: metastore-uri
-                        value_name: METASTORE URI
+                    - index-id:
+                        about: ID of the target index.
+                        long: index-id
+                        value_name: INDEX ID
                         required: true
-                        env: QUICKWIT_METASTORE_URI
                     - index-config-uri:
                         about: Location of the index config file.
                         long: index-config-uri
                         value_name: INDEX CONFIG URI
                         required: true
+                    - index-uri:
+                        about: URI of the target index.
+                        long: index-uri
+                        value_name: INDEX URI
+                    - config-uri:
```

Comment:
nitpick: maybe we could refer to it `quickwit-config-uri` since this is what we are calling it in `.rs` files